.gitignore

```diff
@@ -0,0 +1,7 @@
+src/org/zaproxy/zap/extension/bruteforce/resources/help_*
```

Comment:
This entry should be removed, the files should be tracked. Guess it was added because of build generated JavaHelpSearch files? They are now being properly deleted.